buildSrc/src/main/java/com/r3/csde/CordaLifeCycleHelper.java

```diff
@@ -26,6 +29,8 @@ public void startCorda() throws IOException {
         File combinedWorkerJar = pc.project.getConfigurations().getByName("combinedWorker").getSingleFile();
 
 
+        //docker run -d --rm -p5432:5432 --name CSDEpostgresql -e POSTGRES_DB=cordacluster -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=password postgres:latest
```

Comment:
is this a reference for future understanding or should it be removed as a dead comment ?
If the former - then maybe state this in the comment.

Reply:
Its useful to be able to run the commend manually, I will add comments to that effect.